test/gurka/test_closure_annotations.cc

```diff
@@ -0,0 +1,327 @@
+#include "gurka.h"
```

Comment:
Could we apply more compiler warnings to this file at https://github.com/valhalla/valhalla/blob/master/test/gurka/CMakeLists.txt#L14 ?

Reply:
totally! will add it